install.sh

```diff
@@ -1,42 +1,75 @@
 #!/usr/bin/env bash
 
-read -p "Do you want me to install Group Butler Bot? (Y/N): "
-
-if [ "$REPLY" != "Y" ]; then
-	echo "Exiting..."
-else
-	echo -e "\e[1;36mUpdating packages\e[0m"
-	sudo apt-get update -y
+# Lua Version
+LUA=5.2
 
-	echo -e "\e[1;36mInstalling dependencies\e[0m"
-	sudo apt-get install libreadline-dev libssl-dev lua5.2 luarocks liblua5.2-dev git make unzip redis-server curl libcurl4-gnutls-dev -y
+# Dependencies and libraries
+NATIVE="libreadline-dev libssl-dev luarocks liblua$LUA-dev git make unzip redis-server curl libcurl4-gnutls-dev deborphan"
+ROCKS="luasocket luasec redis-lua lua-term serpent lua-cjson Lua-cURL"
 
-	echo -e "\e[1;36mInstalling LuaRocks from sources\e[0m"
+# Color variables
+Green='\033[0;32m'
+Orange='\033[0;33m'
+Blue='\033[0;34m'
+Cyan='\033[0;36m'
+BRed='\033[1;31m'
+BGreen='\033[1;32m'
+Default='\033[0m'
 
-	git clone http://github.com/keplerproject/luarocks
-	cd luarocks
-	./configure --lua-version=5.2
-	make build
-	sudo make install
-	cd ..
+read -p "Do you want me to install Group Butler Bot? (Y/N): "
 
-	echo -e "\e[1;36mInstalling rocks\e[0m"
+case $REPLY in [yY])
+	# Install Dependencies
+	clear
+	echo -en "${Blue}The packages will be installed:${Default} ${NATIVE}\n${Cyan}Do you want to install the dependencies (Y/N): ${Default}"
+	read REPLY
+	if [[ $REPLY == [yY] ]]; then
+		wget https://www.lua.org/ftp/lua-5.2.4.tar.gz && tar -vzxf lua-5.2.4.tar.gz && cd lua-5.2.4 && sudo make linux test
```

Comment:
I am using the latest 5.2.X update, do not need to use var `$LUA` so much that I tested with it but it gave error, only if I create another variable.
![image](https://user-images.githubusercontent.com/6701156/28020229-f4e6a040-6559-11e7-86df-31d46bb9673b.png)